src/client/js/model/input.js

```diff
@@ -136,7 +136,8 @@ ModelPrototype.setInputFromUrlState = function(rawInput) {
           (allInputsNotInDataset.length > 1 ? 'are' : 'is') +
           ' not in the ' +
           this.database +
-          ' datasets.',
+          ' datasets.' +
+          ' Please select complete animal to view this neuron.',
```

Comment:
C. Nice touch. Does this render OK? It should show up in a translucent red box if I'm not mistaken. How can I trigger it so I can view it?

Reply:
There are a few example neurons in the issue (#105) that trigger this (`M3`, `MI`, `I3`, etc.)